test/api/liskApi.js

```diff
@@ -239,14 +241,14 @@ describe('Lisk.api()', () => {
 
 		it('should list active delegates', () => {
 			const callback = sinon.spy();
-			const options = { limit: '1' };
+			const options = { limit: requestDataLimit };
 			sinon.stub(LSK, 'sendRequest').callsArgWith(3, expectedResponse);
 
-			LSK.listActiveDelegates('1', callback);
+			LSK.listActiveDelegates(requestDataLimit, callback);
 
-			(LSK.sendRequest.calledWith(GET, 'delegates/', options)).should.be.true();
+			(LSK.sendRequest.calledWith(GET, 'delegates', options)).should.be.true();
 			(callback.called).should.be.true();
-			(callback.calledWith(expectedResponse)).should.be.true();
+			(callback.calledWithExactly(expectedResponse)).should.be.true();
```

Comment:
If you specify the callback in the previous `sendRequest` assertion this will become obsolete (also true for other tests below).